src/store/ducks/tests/plans.test.ts

```diff
@@ -1,7 +1,7 @@
 import reducerRegistry from '@onaio/redux-reducer-registry';
 import { cloneDeep, keyBy, keys, pickBy, values } from 'lodash';
 import { FlushThunks } from 'redux-testkit';
-import { REACTIVE, ROUTINE } from '../../../constants';
+import { CASE_TRIGGERED_PLAN, REACTIVE, ROUTINE } from '../../../constants';
```

Comment:
Please use the values from settings.ts, specifically `FIReasons` instead of from constants.  This is so that we keep everything consistent.